src/constants/WalletAndCurrencyConstants.ts

```diff
@@ -131,6 +131,7 @@ type SpecialCurrencyInfo = {
   keysOnlyMode?: boolean
   isPrivateKeySweepable?: boolean
   isBitPayProtocolSupported?: boolean
+  isNotTransactionListSupported?: boolean
```

Comment:
If we are using the is* prefix, best practice is never to do a double negative, makes it hard to read. Especially 'not' because it's what we use to describe the logical '!' operator.




Suggest isHiddenTransactionList

Reply:
I agree with Jon and think we should use `isTransactionListUnsupported`

Reply:
I'll make the change.

Though, it might not entirely solve the problem with double negatives (`!isTransactionListUnsupported` is still a double negative, though a bit easier to read/speak I guess).

Ideally, we could make a convention to never allow negative booleans; we'd have `isTransactionListSupported` and default this to a value of `true` somehow. But I haven't given this approach much thought on how to implement it.